lib/open_api_spex/open_api/decode.ex

```diff
@@ -0,0 +1,402 @@
+defmodule OpenApiSpex.OpenApi.Decode do
```

Comment:
👍 Really like the way this file turned out. Lots of small functions with pipelines.